### ISSUE TITLE
ClassCastException with `<jettyEnvXml>` use in `jetty-ee10-maven-plugin`

### ISSUE THREAD

**Issue by guymac** [eclipse/jetty.project#10440]
I'm upgrading a web-app to Jakarta Servlet 6 / Restful WS 3.1, hence switching from jetty-maven-plugin 9.x to 12.x. With the following Maven configuration snippet, I get an error.			

``` xml
<plugin>
	<groupId>org.eclipse.jetty.ee10</groupId>
	<artifactId>jetty-ee10-maven-plugin</artifactId>
	<version>12.0.0</version>
	<configuration>
		<webApp>
			<jettyEnvXml>${user.home}/.m2/jetty-myapp-env.xml</jettyEnvXml>
		</webApp>
	</configuration>
</plugin>
```

The error is `java.lang.ClassCastException: java.lang.String incompatible with org.eclipse.jetty.util.resource.Resource`. (It doesn't seem to be actually parsing the referenced file, as the same error is given with an invalid path.

**Comment by joakime** [comment]
This is a bug on the maven-ee10-maven-plugin side.

The `<jettyEnvXml>` is a String in the plugin

https://github.com/eclipse/jetty.project/blob/d09a95aa006421c8802e704997fff63d378f9592/jetty-ee10/jetty-ee10-maven-plugin/src/main/java/org/eclipse/jetty/ee10/maven/plugin/MavenWebAppContext.java#L140-L143

And is set as a String to the context attribute

https://github.com/eclipse/jetty.project/blob/d09a95aa006421c8802e704997fff63d378f9592/jetty-ee10/jetty-ee10-maven-plugin/src/main/java/org/eclipse/jetty/ee10/maven/plugin/MavenWebAppContext.java#L321-L322

But the `EnvConfiguration` expects a `Resource`

https://github.com/eclipse/jetty.project/blob/d09a95aa006421c8802e704997fff63d378f9592/jetty-ee10/jetty-ee10-plus/src/main/java/org/eclipse/jetty/ee10/plus/webapp/EnvConfiguration.java#L81-L82

**Comment by janbartel** [comment]
Fixed via #10493
